pyproject.toml

```diff
@@ -0,0 +1,72 @@
+[tool.poetry]
+name = "pyppeteer2"
+version = "0.2.2"
+description = "Headless chrome/chromium automation library (unofficial port of puppeteer)"
+readme = 'README.md'
+license = "MIT"
+homepage = "https://github.com/pyppeteer/pyppeteer2"
```

Comment:
Maybe the homepage should be the docs?

Reply:
We kinda haven't migrated docs yet :grimacing: 
But in the future sure, that would probably be better

Reply:
Alright, leave it for now